include/alpaka/queue/QueueUniformCudaHipRtBlocking.hpp

```diff
@@ -123,11 +123,6 @@ namespace alpaka
                 public:
                     dev::DevUniformCudaHipRt const m_dev;   //!< The device this queue is bound to.
                     ALPAKA_API_PREFIX(Stream_t) m_UniformCudaHipQueue;
-
-#if BOOST_COMP_HCC  // NOTE: workaround for unwanted nonblocking hip streams for HCC (NVCC streams are blocking)
-                    int m_callees = 0;
-                    std::mutex m_mutex;
-#endif
```

Comment:
the workaround is still required. The fix is work in progress though, see [issue update here](https://github.com/ROCm-Developer-Tools/HIP/issues/1205#issuecomment-596009774). Could you run the queue test successfully?
(already wrote this yesterday, but havent saved it)

Reply:
I have not tested it with the latest HIP 3.1 but with the dev version of 3.1. Callbacks are still not working. This workaround was only active for HIP-hcc and for HIP-clang alpaka is not supporting callbacks (for now). Since this PR removes HIP-hcc it is save to remove the workaround.